test/distributed/elastic/rendezvous/dynamic_rendezvous_test.py

```diff
@@ -880,6 +888,8 @@ def setUp(self) -> None:
 
         mock_datetime = self._datetime_patch.start()
         mock_datetime.utcnow.return_value = self._now
+        if device_type == "xpu":
+            mock_datetime.now.return_value = self._now
```

Comment:
May I know why xpu is special, and not reuse `mock_datetime.utcnow.return_value`